openapi.yaml

```diff
@@ -5114,67 +5116,43 @@ components:
         back-end during creation. MUST match the specified pattern.
       pattern: '^[\w\-\.~]+$'
       example: wms-a3cca9
-    argument:
+    resource_parameter:
       x-additionalPropertiesName: Parameter Name
       type: object
-      title: Argument
-      description: Describes a general argument for various entities.
+      title: Resource Parameter
+      description: |-
+        Describes a parameter for various resources (e.g. file formats, service types).
+
+        The parameters are specified according to the [JSON Schema draft-07](http://json-schema.org/) specification.
+        See the chapter ['Schemas' in 'Defining Processes'](#section/Processes/Defining-Processes) for more information.
+
+        It is recommended to refrain from using the following more complex JSON Schema keywords:
+        `if`, `then`, `else`, `readOnly`, `writeOnly`, `dependencies`, `minProperties`, `maxProperties`, `patternProperties`, `multipleOf`.
+
+        JSON Schemas should always be dereferenced (i.e. all `$refs` should be resolved). This allows clients to consume the schemas much better.
```

Comment:
shouldn't the "should" and "recommended" (a bit higher) be in upper case?

Reply:
Yes, indeed. Fixed it. Also made commit bc15fe17 to do some similar changes.